pkg/types/types.go

```diff
@@ -72,6 +72,50 @@ var NodeRoles = map[string]Role{
 	string(LoadBalancerRole): LoadBalancerRole,
 }
 
+// ClusterHealth describes the health of a cluster
+type ClusterHealth string
+
+// possible health states
+const (
+	ClusterHealthOK               ClusterHealth = "OK"                  // all nodes running
+	ClusterHealthDegraded         ClusterHealth = "Degraded"            // 1+ nodes down
+	ClusterHealthOKModified       ClusterHealth = "OK (Modified)"       // some nodes stopped intentionally, rest is running
+	ClusterHealthDegradedModified ClusterHealth = "Degraded (Modified)" // 1+ nodes down and some more stopped intentionally
+	ClusterHealthBroken           ClusterHealth = "Broken"              // all nodes down
```

Comment:
I don't think I understand the value of the distinctions provided by `OK (Modified)` and `Degraded (Modified)`

Reply:
`OK (Modified)` means, that no node crashed, but one or more nodes have been stopped (`k3d node stop`).
However, I'm not sure how to achieve this here, since docker doesn't provide a way of updating labels on existing containers without recreating them and I don't really want to persist any state for k3d itself.
`Degraded (Modified)` means, that some nodes have crashed and some others have been stopped intentionally (`k3d node stop`).

I also have the feeling that this distinction may be a little too granular.